CDBTo3DTiles/src/TileFormatIO.cpp

```diff
@@ -329,14 +328,22 @@ void convertTilesetToJson(const CDBTile &tile, float geometricError, nlohmann::j
         json["content"]["uri"] = *contentURI;
     }
 
-    for (auto child : tile.getChildren()) {
-        if (child == nullptr) {
-            continue;
-        }
+    std::vector<CDBTile*> children = tile.getChildren();
+
+    if(children.empty()) {
```

Comment:
Minor style suggestion. See if you are able to run clang format to auto-format the code. The repo contains a config file in `.clang-format`.

@baothientran how do you normally run clang-format and could those instructions be included in the README?

```suggestion
    if (children.empty()) {
```